examples/ibm-lbaas/main.tf

```diff
@@ -0,0 +1,123 @@
+# Configure the IBM Cloud Provider
+provider "ibm" {}
```

Comment:
Can we add https://github.com/Cloud-Schematics/LAMP/blob/master/provider.tf instead? The idea is to make these easily consumable from IBM Cloud schematics as well.